Lesson27/gozde/update/script.js

```diff
@@ -0,0 +1,53 @@
+const urlParams = new URLSearchParams(window.location.search);
+const postId = urlParams.get('postId');
+
+document.addEventListener('DOMContentLoaded', function() {
+    
+    fetch('https://jsonplaceholder.typicode.com/posts/' + postId)
+  .then((response) => response.json())
+  .then((post) => {
+    document.getElementById('title-input').value = post.title;
+    document.getElementById('body-input').value = post.body;
+    
+}
```

Comment:
There's a closing ')' missing:
```suggestion
})
```